commit 1380ce172e4f98ab433085721f0e8fddef4ba621
Author: Christophe Giboudeaux <cgiboudeaux@gmx.com>
Date:   Thu Apr 28 11:34:39 2011 +0200

    Revert "Maintain sort order when new articles are added"
    
    While fixing the new article sorting issue, this introduced regressions
    when displaying only unread articles.
    
    Alex, please fix and resubmit to kdepim/4.6.
    BUG: 271794
    BUG: 271149
    CCBUG: 271640
    FIXED-IN: kdepim 4.6
    
    This reverts commit e95e0f8e3687137f02e19d3ecf5ea0a9463f5962.

--- a/akregator/src/articlelistview.cpp
+++ b/akregator/src/articlelistview.cpp
@@ -156,11 +156,9 @@
     m_proxy->setSourceModel( model );
     m_proxy->setSortRole( ArticleModel::SortRole );
     m_proxy->setFilters( m_matchers );
-    m_proxy->setDynamicSortFilter( true );
     FilterDeletedProxyModel* const proxy2 = new FilterDeletedProxyModel( model );
     proxy2->setSortRole( ArticleModel::SortRole );
     proxy2->setSourceModel( m_proxy );
-    proxy2->setDynamicSortFilter( true );
 
     FilterColumnsProxyModel* const columnsProxy = new FilterColumnsProxyModel( model );
     columnsProxy->setSortRole( ArticleModel::SortRole );
@@ -169,7 +167,6 @@
     columnsProxy->setColumnEnabled( ArticleModel::FeedTitleColumn );
     columnsProxy->setColumnEnabled( ArticleModel::DateColumn );
     columnsProxy->setColumnEnabled( ArticleModel::AuthorColumn );
-    columnsProxy->setDynamicSortFilter( true );
 
     setModel( columnsProxy );
     header()->setContextMenuPolicy( Qt::CustomContextMenu );
